### DISCUSSION TITLE
test_bench for IP cores

### DISCUSSION THREAD

**Discussion by julianctrt** [srsran/zynq_timestamping#36] [General] [answered]
Hi again,

in my adventure to port the design to the zcu208 I need to debug the modules in hdl, for this reason I would like to know if there are available test bench for the IP cores at least as independent entities.

regards.

**Comment by ofontbach** [comment, marked as answer]
Hi @julianctrt,

we have some code, but honestly it's a little outdated. I've created a [new issue](https://github.com/srsran/zynq_timestamping/issues/37) to add some minimal RTL simulation capabilities to the repo.

Regards
